scripts/execwhacker.py

```diff
@@ -67,6 +67,14 @@ def kill_if_needed(banned_strings_automaton, allowed_patterns, cmdline, pid, sou
                             f"action:spared pid:{pid} cmdline:{cmdline} matched:{b} allowed-by:{ap} source:{source.value}"
                         )
                     return
+            if (
+                    re.search(r"\w" + re.escape(b), cmdline, re.IGNORECASE) or
```

Comment:
Can you add a comment here explaining why we do this?

Reply:
Added